ote_sdk/ote_sdk/entities/annotation.py

```diff
@@ -23,42 +21,17 @@ class Annotation(metaclass=abc.ABCMeta):
     """
 
     # pylint: disable=redefined-builtin;
-    def __init__(
-        self, shape: ShapeEntity, labels: List[ScoredLabel], id: Optional[ID] = None
-    ):
-        self.__id_ = ID(ObjectId()) if id is None else id
+    def __init__(self, shape: ShapeEntity, labels: List[ScoredLabel]):
         self.__shape = shape
         self.__labels = labels
 
     def __repr__(self):
         return (
             f"{self.__class__.__name__}("
             f"shape={self.shape}, "
-            f"labels={self.get_labels(True)}, "
-            f"id={self.id_})"
+            f"labels={self.get_labels(True)}"
```

Comment:
I think for this call and the other calls to get_labels it is nice to include keyword arguments, for example "get_labels(include_empty=True)", because I do not know what the True stands for here.

Reply:
Agreed, didn't really touch it because I was just deleting functionality, but I've addressed the issues in the shapes/annotation/dataset now.